src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1623,4 +1623,10 @@ This version used in: {2}</value>
   <data name="MethodMustReturnStreamThatSupportsReadAndSeek" xml:space="preserve">
     <value>{0} must return a stream that supports read and seek operations.</value>
   </data>
+  <data name="Generate_parameter_0" xml:space="preserve">
+    <value>Generate parameter '{0}'</value>
+  </data>
+  <data name="Generate_parameter_0_and_overrides_implementations" xml:space="preserve">
+    <value>Generate parameter '{0}' (and overrides/implementations)</value>
```

Comment:
❓ Can you add a test showing that this option is not presented in cases where the method is non-virtual and doesn't override or implement another method?

Reply:
TestSimpleWriteCount already does that.